App/makersbnb.rb

```diff
@@ -62,7 +62,8 @@ class Makersbnb < Sinatra::Base
   end
 
   post '/spaces' do
-    Space.create(name: params[:name], description: params[:description], price: params[:price], date_start: params[:date_start],
+    @user = (User.find_by id: session[:user_id])
+    @user.spaces.create(name: params[:name], description: params[:description], price: params[:price], date_start: params[:date_start],
```

Comment:
Is this the correct way to make a space? We need to make an instance of the Space class, right? Let me know if this works

Reply:
all working